### PR TITLE
Issue #7593: Fix typos in docs of TranslationCheck

### PR DESCRIPTION
**Pull request by Mariam-maghraby**
Issue #7593

I did not read this post before submitting the PR.

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheck.java [hunk 1]
```diff
@@ -232,6 +232,7 @@
  *
  * @since 3.0
  */
+
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheck.java:235]: Remove this empty line.

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheck.java [hunk 2]
```diff
@@ -89,18 +89,18 @@
...
- * language code) of required translations the check will also check for existence
- * of default translation files in project.
+ * language code) of required translations, the check will also check for the existence
+ * of default translation files in the project.
  * </p>
  * <p>
  * Attention: the check will perform the validation of ISO codes if the option
  * is used. So, if you specify, for example, "mm" for language code,
- * TranslationCheck will rise violation that the language code is incorrect.
+ * TranslationCheck will rise a violation that the language code is incorrect.
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheck.java:98]: `rise` => `raise`

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheck.java [hunk 3]
```diff
@@ -74,7 +74,7 @@
  * </pre>
  * <p>
  * The Translation check will find the typo in the German {@code hello} key,
- * the missing {@code ok} key in the default resource file and the missing
+ * The missing {@code ok} key in the default resource file and the missing
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheck.java:77]: This is not the start of the statement. We detailing a list with the comma and it is one long sentence. This should be reverted to what it was before.

**Review by rnveach — CHANGES_REQUESTED** [review]
Rename your commit to exactly `doc: fixed typos in docs of TranslationCheck`

**Comment by Mariam-maghraby** [comment]
`Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0:test (default-test) on project checkstyle: There are test failures.` This error pops when I use ` mvn clean verify`

**Comment by Mariam-maghraby** [comment]
@rnveach I rebase it into 1 commit, can you review it now?

**Comment by nrmancuso** [comment]
@Mariam-maghraby review all CI failures and fix issues, example: https://dev.azure.com/romanivanovjr/romanivanovjr/_build/results?buildId=13815&view=logs&j=eb841e9f-6e1e-5c29-3ad5-74ec2c658069&t=a9b761e6-b8d5-50cf-467a-54967189cab2

Content in xdocs and javadoc of check must be synced

**Comment by Mariam-maghraby** [comment]
@nrmancuso on it, thanks!

**Comment by Mariam-maghraby** [comment]
This is the error with maven log but I can't resolve it
image

**Comment by nrmancuso** [comment]
@Mariam-maghraby actual error is above where you took your screenshot. See https://github.com/checkstyle/checkstyle/pull/12918#issuecomment-1486799626 , xdocs and Javadoc must match. You can rerun the failing test in your IDE to see the actual difference.

**Comment by Mariam-maghraby** [comment]
@nrmancuso I don't know how to sync the content in xdocs and javadoc. Can you give me a hint?
Also the build in [Link]((https://dev.azure.com/romanivanovjr/romanivanovjr/_build/results?buildId=13815&view=logs&j=eb841e9f-6e1e-5c29-3ad5-74ec2c658069&t=a9b761e6-b8d5-50cf-467a-54967189cab2))  now is no longer found

**Comment by romani** [comment]
update them manually together

**Comment by Mariam-maghraby** [comment]
> update them manually together

@rnveach Ok got it, thanks!

**Comment by Mariam-maghraby** [comment]
The test run smoothly but how can I resolve these errors?
image

**Comment by rnveach** [comment]
@Mariam-maghraby The number of commits in this PR should be 1, just yours.
https://github.com/Mariam-maghraby/checkstyle
> This branch is [11 commits ahead](https://github.com/Mariam-maghraby/checkstyle/compare/checkstyle:checkstyle:master...master) of checkstyle:master.

See https://github.com/checkstyle/checkstyle/wiki/PR-rules

**Comment by Mariam-maghraby** [comment]
@rnveach I did rebase now

**Comment by Mariam-maghraby** [comment]
what should I do to pass the 2 tests that failed?

**Comment by Mariam-maghraby** [comment]
Is there a way instead of manually to figure out at which line or paragraph tag that the xdocs anen't synced with the javadocs?

**Comment by rnveach** [comment]
> Is there a way instead of manually to figure out at which line or paragraph tag that the xdocs anen't synced with the javadocs?

There is not.

CI has checkstyle failures which give line numbers.
https://app.circleci.com/pipelines/github/checkstyle/checkstyle/18172/workflows/ece7a0ce-036c-454c-b1f0-9f0bef295546/jobs/278269
````
[INFO] [checkstyle] Running Checkstyle  on 1499 files
[ERROR] [checkstyle] [ERROR] /root/project/src/main/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheck.java:232: Javadoc tag '@since' should be preceded with an empty line. [RequireEmptyLineBeforeBlockTagGroup]
[ERROR] [checkstyle] [ERROR] /root/project/src/xdocs/config_misc.xml:1: File does not end with a newline. [NewlineAtEndOfFile]
[ERROR] [checkstyle] [ERROR] /root/project/src/xdocs/config_misc.xml:2240: Line should not be longer than 100 symbols [lineLengthXml]
[INFO]      [echo] Checkstyle finished (checkstyle-checks.xml) : 11/04/2023 07:06:40 PM
[INFO]      [echo] Checkstyle started (checkstyle-non-main-files-checks.xml): 11/04/2023 07:05:44 PM
[INFO] [checkstyle] Running Checkstyle  on 3559 files
[ERROR] [checkstyle] [ERROR] /root/project/src/xdocs/config_misc.xml:1: File does not end with a newline. [NewlineAtEndOfFile]
[ERROR] [checkstyle] [ERROR] /root/project/src/xdocs/config_misc.xml:2240: Line should not be longer than 100 symbols [lineLength]
````

Also, you need to rebase now as there are conflicts.

**Comment by Mariam-maghraby** [comment]
image

**Comment by rnveach** [comment]
I am sorry, I have not seen that before. If you are using IntelliJ, maybe another member or google can help you. I only use Eclipse.

**Comment by romani** [comment]
Please backup your changes, close Pr. Remove your fork. And do all from scratch.
https://checkstyle.org/beginning_development.html#Starting_Development

**Comment by Mariam-maghraby** [comment]
@romani Ok, I will do so thanks!

**Pull request closed without merging**